en/core-api/advanced-features.md

```diff
@@ -126,16 +130,95 @@ curl -X POST \
+For general use, the JSON parameters recommended to be included in the Charge API request are described in the table given below.
+
+<table style="width:100%">  
+  <tr>     
+    <th colspan="2">Name</th> <th>Description</th>  <th>Type</th>  
+  </tr>   
+    <tr>      
+    <td>item_details</td> <td></td> <td>The details of the item purchased.</td> <td>Object</td>
+  </tr>
+  <tr>      
+   <td></td> <td>id</td>  <td>The id of the item purchased.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>price</td>  <td>The price of the item.</td> <td>String</td>
+  </tr>
+   <tr>      
+   <td></td> <td>quantity</td>  <td>The number of items purchased.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>name</td>  <td>The name of the item purchased.</td> <td>String</td>
+  </tr>
+  <tr>      
+    <td>customer_details</td> <td></td> <td>The details of the customer.</td> <td>Object</td>
+  </tr>
+  <tr>      
+   <td></td> <td>first_name</td>  <td>The first name of the payer. </td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>last_name</td>  <td>The last name of the payer. </td> <td>String</td>
+  </tr>
+   <tr>      
+   <td></td> <td>email</td>  <td>The email address of the payer. </td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>phone</td>  <td>The phone number of the payer.</td> <td>String</td>
+    </tr> <tr>      
+   <td></td> <td>address</td>  <td>The postal address of the payer.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>city</td>  <td>The city of payer.</td> <td>String</td>
+  </tr>
+    <tr>      
+   <td></td> <td>postal_code</td>  <td>The postal code of the payer.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>country_code</td>  <td>The country code of the payer. </td> <td>String</td>   </tr> <td>shipping_address</td> <td></td> <td>The address to which the item is to be shipped.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>first_name</td>  <td>The first name of the customer.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>last_name</td>  <td>The last name of the customer</td> <td>String</td>
+  </tr>
+   <tr>      
+   <td></td> <td>email</td>  <td>The email address of the customer.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>phone</td>  <td>The phonenumber of the customer.</td> <td>String</td> </tr><tr>      
+   <td></td> <td>address</td>  <td>The postal address.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>city</td>  <td>The name of the city.</td> <td>String</td>
+  </tr>
+   <tr>      
+   <td></td> <td>postal_code</td>  <td>The postal code.</td> <td>String</td>
+  </tr>
+  <tr>      
+   <td></td> <td>country_code</td>  <td>The country code.</td> <td>String</td>   </tr></table>
+For more details, refer to [JSON Object](https://api-docs.midtrans.com/#json-object).
```

Comment:
The link didn't get rendered properly, you can try to add new blank line after the table. So the next paragraph line is not considered as HTML, and will be considered as markdown